src/widgets/clear-all/clear-all.js

```diff
@@ -90,6 +95,12 @@ function clearAll({
         />,
         containerNode
       );
+    },
+
+    clearAll() {
+      if (this.clearAttributes.length > 0) {
```

Comment:
The issue is that in order to protect certain attributes from clearing, an attribute list must be passed to `clearRefinementsAndSearch`.  If there are no clearable attributes then nothing should happen  even if the caller overrides autohideContainer